src/Components/Game/Game.js

```diff
@@ -45,15 +45,16 @@ import ErrorDisplay from "../ErrorDisplay/ErrorDisplay"
       setDeckID(data.data.createDeck.deck.id)
     })
     .catch(err => {
-      console.log(err)
-      setError(err)})
+      console.log('error', err)
+      setError(true)})
   }, [])
   
   return (
     <div className="question-container">
-      {error 
+      {/* {error 
       ? <ErrorDisplay errorCode='500' /> 
-      : deckID ? <Question deckID={deckID} difficulty={location[2]}/> : <Loading/>}
+      :  */}
+      {deckID ? <Question deckID={deckID} difficulty={location[2]}/> : <Loading/>}
     </div>
```

Comment:
We need to uncomment the error display line. It was initially commented out while we tested for a bug.